templates/arc-ts/app/entry.client.tsx

```diff
@@ -1,4 +1,4 @@
 import { hydrate } from "react-dom";
-import { RemixBrowser } from "remix";
+import { RemixBrowser } from "@remix-run/react";
```

Comment:
Is this really what we want?

One of the advertised/suggested things was
> just import everything from the `remix` package

Now it seems like this isn't possible anymore?

CC/ @mjackson @ryanflorence

Reply:
Yes, @MichaelDeBoey. The purpose of this PR is to better define the server runtime packages and move away from importing everything from `remix`.

The `remix` package has a few downsides, not the least of which is that it requires a `remix setup`. It's also non-standard behavior to change things in `node_modules` after they are installed. This will improve compatibility with the rest of the node ecosystem, including alternative package managers like yarn plug'n'play and pnpm.